src/lqcut.jl

```diff
@@ -127,23 +118,23 @@ function lq_drift_time_correction(
 
     return result, report
 end
-export lq_drift_time_correction
+export lq_ctc_correction
 
 """
-    DEP_µ::Unitful.Energy, DEP_σ::Unitful.Energy, e_cal::Vector{<:Unitful.Energy}, lq_classifier::Vector{Float64}; cut_sigma::Float64=3.0, truncation_sigma::Float64=2.0)
+    DEP_µ::Unitful.Energy, DEP_σ::Unitful.Energy, e_cal::Vector{<:Unitful.Energy}, lq_classifier::Vector{<:AbstractFloat}; cut_sigma::Float64=3.0, dep_sideband_sigma::Float64=4.5, cut_truncation_sigma::Float64=2.0)
 
 Evaluates the cutoff value for the LQ cut. The function performs a binned gaussian fit on the sidebandsubtracted LQ histogram and evaluates the cutoff value difined at 3σ of the fit.
 # Returns
     * `result`: NamedTuple of the cutoff value
     * `report`: NamedTuple of the fit result, fit report and temporary histograms
 """
-function LQ_cut(
-    DEP_µ::Unitful.Energy, DEP_σ::Unitful.Energy, e_cal::Vector{<:Unitful.Energy}, lq_classifier::Vector{<:AbstractFloat}; cut_sigma::Float64=3.0, truncation_sigma::Float64=2.0)
+function lq_cut(
+    DEP_µ::Unitful.Energy, DEP_σ::Unitful.Energy, e_cal::Vector{<:Unitful.Energy}, lq_classifier::Vector{<:AbstractFloat}; cut_sigma::Float64=3.0, dep_sideband_sigma::Float64=4.5, cut_truncation_sigma::Float64=2.0)
 
     # Define sidebands
-    lq_DEP = lq_classifier[DEP_µ - 4.5 * DEP_σ .< e_cal .< DEP_µ + 4.5 * DEP_σ]
-    lq_sb1 = lq_classifier[DEP_µ -  2 * 4.5 * DEP_σ .< e_cal .< DEP_µ - 4.5 * DEP_σ]
-    lq_sb2 = lq_classifier[DEP_µ + 4.5 * DEP_σ .< e_cal .< DEP_µ + 2 * 4.5 * DEP_σ]
+    lq_DEP = lq_classifier[DEP_µ - dep_sideband_sigma * DEP_σ .< e_cal .< DEP_µ + dep_sideband_sigma * DEP_σ]
+    lq_sb1 = lq_classifier[DEP_µ -  2 * dep_sideband_sigma * DEP_σ .< e_cal .< DEP_µ - dep_sideband_sigma * DEP_σ]
+    lq_sb2 = lq_classifier[DEP_µ + dep_sideband_sigma * DEP_σ .< e_cal .< DEP_µ + 2 * dep_sideband_sigma * DEP_σ]
```

Comment:
Could you somehow make the casing (uppercase/lowercase) of `DEP`/`dep` consistent?

Reply:
Would you suggest in variables lower case for all variables, except the "peak_names"? Because there they usually appear uppercase

Reply:
Whatever choice you take, it should be consistent.

Reply:
ok then I will write dep in variables lowercase, because all variables are typically lowercase,but when talking about it e.g in docstrings have the abbreviation uppercase